### ISSUE TITLE
Date numbers not translating into Farsi numerics 

### ISSUE THREAD

**Issue by baghalipolo** [NUKnightLab/TimelineJS#800]
When you change the language to Farsi, the month shows up in Farsi but numbers (i.e. year and date) show up in English numerics. When I pasted Farsi numerics into the google spreadsheet the website preview gets stuck on "building dates". I assume this is because it does not recognize it. 

Another issue is that when you select the date from the calendar in the spreadsheet, it translates into the incorrect month because the Iranian calendar is different (e.g. we are currently in the 3rd month of the Iranian calendar, not the 6th). 

I'm mostly concerned with the numeric translation, is there a way to ensure full translation through the google spreadsheet format?

**Comment by JoeGermuska** [comment]
At this time, TimelineJS cannot process non-arabic numerals, nor can we support non-Gregorian calendars.

We are hard at work on a rewrite to Timeline (http://timeline3.knightlab.com) which is the focus of our future development. One of its features is support for a "display date" so that you can override how a date is shown on a given event, which would be a way to present Farsi numerals. However, I'm afraid at this time we have not considered how to support that kind of control over the labels on the bottom of the navigation panel.

It's even harder to know how we will handle non-Gregorian calendars. We have logged it as a feature to be desired (NUKnightLab/TimelineJS3#83) but we have much to learn before we can even estimate the effort it will take to support them.

Since we are shifting our development focus to the new version of Timeline, I am going to close this ticket. A substantial update to the new timeline3.knightlab.com site will be published within the week, including support for creating timelines using a Google spreadsheet which supports the "display date".
